tools/linter/adapters/pyfmt_linter.py

```diff
@@ -62,8 +62,9 @@
                     "torch/d*/**",
                     # torch/[e-n]*/**
                     "torch/[e-n]*/**",
-                    # torch/[o-z]*/**
-                    "torch/[o-z]*/**",
+                    "torch/optim/**",
```

Comment:
```suggestion
                    # torch/optim/**
                    "torch/optim/**",
```

Add a line above to allow `torch/[e-n]*` and `torch/optim` can be migrated separately.

Reply:
Done